README.md

```diff
@@ -19,6 +19,7 @@ directory of a git repo:
 git open [-help]
          [-branch=<branch> | -b=<branch> | -branch | -b]
          [-commit=<SHA> | -c=<SHA> | -commit | -c]
+         [-compare=<branch> | -co=<branch> | -compare | -co]
```

Comment:
Would you be able to add `base` to the options here as we? i.e. `-compare=<branch-name>:<base-name>`